### PR TITLE
[Mailer][Infobip] Document the usage of custom headers

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | -
| License       | MIT

See https://github.com/symfony/symfony-docs/pull/18169/

### CODE REVIEW

**Comment by ndousson** [comment]
Thanks for taking in into account 🙏🏻 

Do you want to tackle the 3 new ones from this MR too ? 👇🏻 
https://github.com/symfony/symfony/pull/57424

These 3 new one will only be available from 7.2. I can create the related MR is you want

**Comment by javiereguiluz** [comment]
Once this PR is merged, you can create a new one for the 7.2 branch to document the three new headers. Thanks!

File: src/Symfony/Component/Mailer/Bridge/Infobip/README.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ MAILER_DSN=infobip+api://KEY@BASE_URL
...
+| -------------------------------- | ------- | -------------------------------------------------------------------------------------------- |
+| `X-Infobip-IntermediateReport`   | boolean | The real-time Intermediate delivery report that will be sent on your callback server.        |
+| `X-Infobip-NotifyUrl`            | string  | The URL on your callback server on which the Delivery report will be sent.                   |
+| `X-Infobip-NotifyContentType`    | string  | Preferred Delivery report content type. Can be application/json or application/xml.          |
+| `X-Infobip-MessageId`            | string  | The ID that uniquely identifies the message sent to a recipient.                             |
+| `X-Infobip-Track`                | boolean | Enable or disable open and click tracking.                                                   |
+
 Resources
 ---------
 * [Infobip Api Docs](https://www.infobip.com/docs/api#channels/email)
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Infobip/README.md]:
```suggestion
 * [Infobip Api Docs](https://www.infobip.com/docs/api#channels/email)
```

same for the following lines

**Comment by nicolas-grekas** [comment]
Thank you @javiereguiluz.

**Pull request merged**